R/read_output.R

```diff
@@ -420,7 +419,7 @@ read_saturation_indices <- function(txt) {
 
  names(txt_clean) <- txt_header
```

Comment:
What about the header? Instead of log_k_<temp>_<pressure> it should be only `log_k` afterwards....
https://github.com/KWB-R/geosalz.phreeqc/issues/8#issue-1274776388

Reply:
Ah ok it is already fixed. Ran locally pkgdown::build_site()

![grafik](https://user-images.githubusercontent.com/11964451/174283188-159bcc08-7f11-4be7-a062-95dd092c159b.png)